Week1/databases/index.js

```diff
@@ -0,0 +1,128 @@
+var mysql = require('mysql');
+var connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'hyfuser',
+  password: 'hyfpassword',
+  database: 'userdb',
+});
+
+function handleErrorAndData(error, data) {
+  if (error) {
+    console.log(error);
+  } else {
+    console.log(data);
+  }
+}
+
+connection.connect(error => handleErrorAndData(error, 'MySQL is connected'));
+
+connection.query('CREATE DATABASE IF NOT EXISTS world', error =>
```

Comment:
You can use 'new_world' instead of 'world' here to use the data being loaded by world.sql